src/index.html

```diff
@@ -10,7 +10,21 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body data-qa="hover">
+    <!--First bar-->
+    <form class="search-bar" data-qa="big">
+      <input
+        class="search-bar__text"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
```

Comment:
```suggestion
      <input
        class="search-bar__text"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```

Reply:
That is correct formatting - it's better to stick to it